hack/preview.sh

```diff
@@ -97,6 +97,8 @@ fi
 [ -n "${JVM_BUILD_SERVICE_SIDECAR_IMAGE_TAG}" ] && yq -i e "(.images.[] | select(.name==\"run-maven-component-build\")) |=.newTag=\"${JVM_BUILD_SERVICE_SIDECAR_IMAGE_TAG}\"" $ROOT/components/build/jvm-build-service/kustomization.yaml
 [ -n "${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_REPO}" ] && yq -i e "(.images.[] | select(.name==\"lookup-artifact-location\")) |=.newName=\"${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_REPO}\"" $ROOT/components/build/jvm-build-service/kustomization.yaml
 [ -n "${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_TAG}" ] && yq -i e "(.images.[] | select(.name==\"lookup-artifact-location\")) |=.newTag=\"${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_TAG}\"" $ROOT/components/build/jvm-build-service/kustomization.yaml
```

Comment:
and the same with this line

your added line 100 is the correct way to inject this image ... these 2 lines are incorrect / have not affect